Libraries/VtkVgCore/vtkVgPlotTimeline.cxx

```diff
@@ -232,7 +232,7 @@ struct IdToPointXIter
     : Points(pts), Pointer(start)
     {}
 
-  double operator*()
+  double operator*() const
```

Comment:
(FYI, it looks like this was also fixed by ace613f79b6a52f0404a1fa7e6cac105e602bba4.)